lib/balance-tx/lib/internal/Internal/Cardano/Write/Tx/Balance.hs

```diff
@@ -913,6 +884,13 @@ balanceTransactionWithSelectionStrategyAndNoZeroAdaAdjustment
             SNothing -> return ()
             SJust _ -> throwE ErrBalanceTxExistingReturnCollateral
 
+data Selection = Selection
```

Comment:
Could have a corresponding `applySelection :: Selection -> Tx -> Tx`, which would be `updateTx` but without the `feeUpdate`.